frontend/site/donnees_de_forte_valeur.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Suivi de l'ouverture des données de forte valeur"
+titleTemplate: ':title'
+---
+
+<script setup>
+import HighValueDatasets from './.vitepress/theme/components/HighValueDatasets.vue';
+</script>
+
+# Bienvenue sur le tableau de suivi des ouvertures
+
+Ce tableau permet de suivre et de rendre compte de l’ouverture des jeux de données, algorithmes, codes sources et API publics.
```

Comment:
We'll need the right wording here as well @agarrone

Reply:
Done in 1e93b17